### PR TITLE
Feature/extend ticket with related entities

### PR DESCRIPTION
**Pull request by Bogomazz**

### CODE REVIEW

File: HdProduction.HelpDesk.Api/Controllers/TicketsController.cs [hunk 1]
```diff
@@ -11,26 +13,46 @@ namespace HdProduction.HelpDesk.Api.Controllers
...
+    public async Task<TicketResponseModel> Create(TicketRequestModel requestModel)
+    {
+      var id = await _ticketService.CreateAsync(new Ticket(requestModel.Issue, requestModel.Details, requestModel.IssuerEmail));
+      return await Get(id); 
+    }
+    
+    [HttpPut("{id}")]
+    public async Task<TicketResponseModel> Update(long id, TicketRequestModel requestModel)
+    {
+      await _ticketService.UpdateAsync(id);
```
Comment (farexy) [inline HdProduction.HelpDesk.Api/Controllers/TicketsController.cs:47]: nice:) ok, let's suppose it is for better times

File: HdProduction.HelpDesk.Api/Controllers/TicketsController.cs [hunk 2]
```diff
@@ -11,26 +13,46 @@ namespace HdProduction.HelpDesk.Api.Controllers
...
-      return _mapper.Map<TicketAdminResponseModel>(await _ticketsRepository.FindAsync(id));
+      return _mapper.Map<TicketResponseModel>(await _ticketService.FindAsync(id));
     }
-
+    
     [HttpPost("")]
-    public async Task<TicketAdminResponseModel> Create(TicketRequestModel requestModel)
+    public async Task<TicketResponseModel> Create(TicketRequestModel requestModel)
+    {
+      var id = await _ticketService.CreateAsync(new Ticket(requestModel.Issue, requestModel.Details, requestModel.IssuerEmail));
```
Comment (farexy) [inline HdProduction.HelpDesk.Api/Controllers/TicketsController.cs]: we've already discussed, pass parameters to service, not domain model

File: HdProduction.HelpDesk.Api/Models/Tickets/TicketResponseModel.cs [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
 namespace HdProduction.HelpDesk.Api.Models.Tickets
 {
-  public class TicketResponseModel : TicketRequestModel
-  {
-    
-  }
+    public class TicketResponseModel : TicketRequestModel
+    {
+        public long Id { get; set; }
+        public TicketAttribute TicketStatus {get;set;}
```
Comment (farexy) [inline HdProduction.HelpDesk.Api/Models/Tickets/TicketResponseModel.cs]: 1st - why do you use type TicketAttribute for properties, if it is possible to use concrtete type (TicketStatus, TicketPriority etc.)?
2nd - ok, if you use it like TicketAttribute, than use TicketAttributeResponse model at least

File: HelpDesk.App.Global/appsettings.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
   },
   "AllowedHosts": "*",
   "RsaKeysPath": {
-    "Public": "D:\\BOAuthRsaKeys\\HdProduction\\public.json",
-    "Private": "D:\\BOAuthRsaKeys\\HdProduction\\private.json"
+    "Public": "/Users/aboho/Desktop/hd-production/dashboard-api/HdProduction.Dashboard.Api/RSAKeys/public.json",
```
Comment (farexy) [inline HelpDesk.App.Global/appsettings.json]: you are pidor
never do like this(
create your own env or just not commit that

Comment (Bogomazz) [reply HelpDesk.App.Global/appsettings.json]: Sorry 😅
It shouldn't have gotten into git

Comment (farexy) [reply HelpDesk.App.Global/appsettings.json]: ok, you've added your env.. but what about changes in this file??

File: HelpDesk.App.PostgresSql/appsettings.macbook.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "ConnectionStrings": {
```
Comment (farexy) [inline HelpDesk.App.PostgresSql/appsettings.macbook.json:2]: actually, you do not need to copy whole appsettings file, just things that are different from appsettings.json, such as RsaKeysPath

**Pull request merged**